en/Formulaic Content Policies

```diff
@@ -0,0 +1,119 @@
+We may moderate content shared on the Formulaic platform, to maintain a healthy community and a minimum quality standard.  Our moderation will be based on the following rules.
```

Comment:
Don't we need a title and a date for the document?

Reply:
Yes - can we add today's date?

Reply:
I'm going to add `Formulaic Content Policies` for now, we can update later if needed.